api/hearings/hearing.mock.ts

```diff
@@ -68,7 +68,7 @@ export const init = () => {
   mock.onPut(updateHearingRequest).reply(() => {
     return [
       200,
-      [],
+      HEARING_ACTUAL,
```

Comment:
in the API contract it seems it only returns status code but not any obj.

Reply:
I have updated the code.